### PR TITLE
Generator for Avo

### PR DESCRIPTION
**Pull request by mausamp**

### CODE REVIEW

File: test/generators/avo_install_generator_test.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_avo_gem_successfully
```
Comment (coolprobn) [inline test/generators/avo_install_generator_test.rb]: Let's remove "successfully" since install will either be success or fail with error, this word is just redundant due to that.

File: test/generators/avo_install_generator_test.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    Dir.chdir(app_path) do
+      quietly { run_generator }
+
+      assert_gem 'avo'
+    end
+  end
+
+  def test_should_configure_avo_gem_successfully
```
Comment (coolprobn) [inline test/generators/avo_install_generator_test.rb]: Same as above

File: test/generators/avo_install_generator_test.rb (hunk 1 above)
Comment (coolprobn) [inline test/generators/avo_install_generator_test.rb]: Let's update this to "test_should_configure_avo_gem" and merge tests from "test_should_configure_avo_gem_successfully" here as well since we don't have any options to check for OR edge cases to look out for this to be multiple test_ blocks.

**Review by coolprobn — CHANGES_REQUESTED** [review]

**Review by coolprobn — APPROVED** [review]
